### DISCUSSION TITLE
Very buggy layer showcase

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#5484] [General]
The showcase of the different layers under the social cards' "Customization" section is very buggy, especially when hovering over it and trying to focus on one layer alone.

Perhaps it would be better to have it just be a sort of navigation, where you can press buttons to add/remove layers and show/hide the final result.

**Comment by squidfunk** [comment]
Could you clarify what you mean by "very buggy"?

**Reply by Andre601** [reply]
Some strange jittering or whatever you would call this for the smallest movement of the cursor.

Here's a recording showing that
https://user-images.githubusercontent.com/11576465/236912105-1da0cec7-5dd7-435d-8ec4-36c7b224bf2c.mp4

**Reply by squidfunk** [reply]
Thanks for the screencast. That is really weird. I'll check if I can improve the visualization.

**Reply by kamilkrzyskow** [reply]
Can confirm the same happens on my Firefox too + this visualization doesn't work well on mobile Firefox either. 
Would work better with up/down arrow buttons on the side or simply content tabs.
